src/main/resources/db/migration/V48__ulest-hendelse.sql

```diff
@@ -0,0 +1,12 @@
+create table ulest_hendelse
+(
+    id                uuid                                               not null primary key,
+    deltaker_id       uuid                                               not null references deltaker,
```

Comment:
Kan det skje at vi mottar hendelse på en deltaker som ikke er lagret? isåfall vil det kunne skape problemer med denne constrainten. Lurer på om det kan være bra nok å droppe hele denne contrainten også, for å unngå samtidighetsproblemer